sdk/provisioning/Azure.Provisioning.CloudMachine/src/AzureSdkExtensions/OpenAiModelDeployment.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+public class OpenAiModelDeployment
+{
+    public OpenAiModelDeployment(string model, string modelVersion) { Model = model; ModelVersion = modelVersion; }
+    public string Model { get; }
+    public string ModelVersion { get; }
```

Comment:
maybe we also need a property to specify if the model is for chat, embeddings, etc.